src/api/views.py

```diff
@@ -116,3 +118,72 @@ def oai(request):
     }
 
     return render(request, template, context, content_type="application/xml")
+
+
+def kbart_tsv(request):
+    return kbart(request, delim='\t')
+
+
+def kbart(request, delim=','):
+    """
+    Produces KBART metadata output according to the spec:
+    https://doi.org/10.1080/0361526X.2017.1309826
+    @param request: the request object
+    @param delim: whether to use csv (',') or tsv ('\t') deliminators
+    @return: a rendered CSV or TSV
+    """
+    fields = ["date_first_issue_online", "num_first_vol_online",
+              "num_first_issue_online", "date_last_issue_online",
+              "num_last_vol_online", "num_last_issue_online",
+              "title_url", "first_author", "title_id", "embargo_info",
+              "coverage_depth", "notes", "publisher_name", "publication_type",
+              "date_monograph_published_print",
+              "date_monograph_published_online", "monograph_volume",
+              "monograph_edition", "first_editor",
+              "parent_publication_title_id", "preceding_publication_title_id",
+              "access_type"]
+
+    response = HttpResponse(content_type='text/csv')
+
+    writer = csv.DictWriter(response, fieldnames=fields, delimiter=delim)
```

Comment:
Using a regular writer + an ordered dict saves us from maintaining two sets of keys (`fields` + each key added to the dictionary. It also makes the absence of a value for a key explicit rather than implicit (e.g. did Martin forget to add `"first_editor"` or has he left it out on purpose)

`journal_dict["first_editor"] = None # We don't support this because ABCXYZ`

Reply:
Done (except maintained OrderedDict with late initialisation)